src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -273,6 +273,7 @@ export default function V2ProjectReconfigureModal({
         onConfirmed() {
           setReconfigureTxLoading(false)
           onOk()
+          window.location.reload()
```

Comment:
**suggestion** thinking a better place to put this might be in the `onOk` prop itself. WDYT?

https://github.com/jbx-protocol/juice-interface/blob/f90b3c1ffeec5b4bf12aa0a8211ff8fcab245647/src/components/v2/V2Project/V2ProjectReconfigureModal/V2ReconfigureModalTrigger.tsx#L129

Reply:
done